conf/xml/RobotStateProvider_iCub3.xml

```diff
@@ -124,20 +124,16 @@
         <action phase="shutdown" level="5" type="detach"/>
     </device>
 
-    <device type="human_state_publisher" name="RobotStatePublisher">
-        <param name="period">0.02</param>
-        <param name="baseTFName">/iCub/root_link</param>
-        <param name="humanJointsTopic">/iCub/joint_states</param>
-	      <param name="portprefix">icub</param>
-	      <param name="basePositionOffset">(0.0 3.0 0.0)</param>
-       	<!--param name="baseOrientationOffset">(0.7071068 0.0 0.0 0.7071068)</param-->
-        <action phase="startup" level="5" type="attach">
-            <paramlist name="networks">
-                <elem name="HumanStatePublisherLabel">RobotStateProvider</elem>
-            </paramlist>
-        </action>
-        <action phase="shutdown" level="5" type="detach"/>
-    </device>
+<device type="human_state_wrapper" name="HumanStateWrapper">
+    <param name="period">0.02</param>
+    <param name="outputPort">/HDE/HumanStateWrapper/state:o</param>
+    <action phase="startup" level="5" type="attach">
+        <paramlist name="networks">
+            <elem name="HumanStateWrapperLabel">RobotStateProvider</elem>
+        </paramlist>
+    </action>
+    <action phase="shutdown" level="5" type="detach"/>
+</device>
```

Comment:
There is already a `HumanStateWrapper` in this file.

Reply:
I am removing the changes in this file, so resolving this comment.